venues/ICLR.cc/2019/process/submissionProcess.js

```diff
@@ -0,0 +1,46 @@
+function() {
+  var or3client = lib.or3client;
+  console.log('submission process');
+
+  var SHORT_PHRASE = 'ICLR 2019';
+  var CONFERENCE_ID = 'ICLR.cc/2019/Conference';
+  var PROGRAM_CHAIRS_ID = CONFERENCE_ID + '/Program_Chairs';
+  var REVIEWERS_ID = CONFERENCE_ID + '/Reviewers';
+  var AREA_CHAIRS_ID = CONFERENCE_ID + '/Area_Chairs';
+  var BLIND_SUBMISSION_ID = CONFERENCE_ID + '/-/Blind_Submission';
+
+  var PAPER_GROUP_ID = CONFERENCE_ID + '/Paper' + note.number;
+  var PAPER_AUTHORS_ID = PAPER_GROUP_ID + '/Authors';
+
+  console.log('constants declared');
+
+  var revisionInvitation = {
+    id: CONFERENCE_ID + '/-/Paper' + note.number + '/Add_Revision',
+    readers: ['everyone'],
+    writers: [CONFERENCE_ID],
+    invitees: [],
+    signatures: [CONFERENCE_ID],
+    duedate: invitation.duedate,
+    reply: {
+      referent: note.id,
+      forum: note.forum,
+      content: invitation.reply.content,
+      signatures: invitation.reply.signatures,
+      writers: invitation.reply.writers,
+      readers: invitation.reply.readers
+    }
+  };
+  or3client.or3request(or3client.inviteUrl, revisionInvitation, 'POST', token)
```

Comment:
We can set all the authorids as writers of the note, so there is no need to create an add revision invitation